src/main/java/org/assertj/core/error/ShouldHaveAllNullFields.java

```diff
@@ -0,0 +1,48 @@
+package org.assertj.core.error;
+
+import java.util.List;
+
+/**
+ * Creates an <code>{@link AssertionError}</code> indicating that an assertion that verifies that an object
+ * has null fields failed.
+ *
+ * @author Vladimir Chernikov
+ */
+public class ShouldHaveAllNullFields extends BasicErrorMessageFactory {
+
+  private static final String EXPECTED_MULTIPLE = "%nExpecting%n  <%s>%nto have properties or fields named <%s> are null.%n";
```

Comment:
replace by `"%nExpecting%n  <%s>%nto only have null properties or fields but these were not null:%n  <%s> are null.%n";`

Reply:
Changed.